openapi/connector_mgmt.yaml

```diff
@@ -1719,6 +1735,17 @@ components:
         error_handling:
           dead_letter_queue:
             topic: "dlq"
+        service_account:
+            id: "1"
+            kind: "ServiceAccount"
+            href: "/api/connector_mgmt/v1/serviceaccounts/1"
```

Comment:
@valdar openapi spec looks for a service account here. currently there isn't any endpoint for `/api/connector_mgmt/v1/serviceaccounts/1` I'm hesitant to populate with a non functional example. I'm unsure what way to proceed though, do you have any opinion?